client/src/featureform/dashboard_metadata.py

```diff
@@ -21,9 +22,11 @@
 import os
 from featureform import ResourceClient
 from .version import get_package_version
+from featureform.serving import LocalClientImpl
 
 
 path = os.path.join(os.path.dirname(__file__), "dashboard")
+localClientImpl = LocalClientImpl()
```

Comment:
any reason why this is global and not just inside the function below?

Reply:
figured it would be expensive to run on each request, but if it's negligible I'll make it local.